src/components/income/W2JobInfo.tsx

```diff
@@ -1,57 +1,13 @@
 import React, { ReactElement, useState } from 'react'
 import { useForm } from 'react-hook-form'
-import { Avatar, Button, IconButton, List, ListItem, ListItemAvatar, ListItemSecondaryAction, ListItemText } from '@material-ui/core'
 import { useDispatch, useSelector } from 'react-redux'
-import { Actions, addW2, removeW2 } from '../../redux/actions'
 import { PagerContext } from '../pager'
 import { TaxesState, IncomeW2, Person, PersonRole } from '../../redux/data'
-import DeleteIcon from '@material-ui/icons/Delete'
 import { Currency, formatSSID, GenericLabeledDropdown, LabeledInput } from '../input'
 import { Patterns } from '../Patterns'
-import FormContainer from '../FormContainer'
-
-interface W2ListItemProps {
-  w2: IncomeW2
-  remove: () => void
-}
-
-const W2ListItem = ({ w2, remove }: W2ListItemProps): ReactElement => (
-  <ListItem>
-    <ListItemAvatar>
-      <Avatar>
-      </Avatar>
-    </ListItemAvatar>
-    <ListItemText
-      primary={w2.occupation}
-      secondary={<span>Income: <Currency value={w2.income} /></span>}
-    />
-    <ListItemSecondaryAction>
-      <IconButton onClick={remove} edge="end" aria-label="delete">
-        <DeleteIcon />
-      </IconButton>
-    </ListItemSecondaryAction>
-  </ListItem>
-)
-
-function ListW2s (): ReactElement {
-  const w2s = useSelector((state: TaxesState) =>
-    state.information.w2s
-  )
-
-  const dispatch = useDispatch()
-
-  const drop = (i: number): Actions => dispatch(removeW2(i))
-
-  return (
-    <List dense={true}>
-      {
-        w2s.map((w2, i) =>
-          <W2ListItem key={i} remove={() => drop(i)} w2={w2} />
-        )
-      }
-    </List>
-  )
-}
+import { FormListContainer } from '../FormContainer'
+import { Work } from '@material-ui/icons'
+import { addW2, editW2 } from '../../redux/actions'
```

Comment:
Should have the ability to remove W2 as well. Looks like that ability was removed for W2s in this PR 
![image](https://user-images.githubusercontent.com/8848557/116839584-c2ad3d00-ab87-11eb-8f2b-75748aa569dd.png)

Reply:
Ok got that.